app/src/main/java/com/navelfuzz/taskmaster/MainActivity.java

```diff
@@ -46,14 +57,14 @@ protected void onCreate(Bundle savedInstanceState) {
         setupSettingsButton();
         setupAddTaskButton();
         setupAllTasksButton();
+        updateTasksListFromDatabase();
```

Comment:
only required in the onResume since onResume will always run right after onCreate. May need to move `setupRecyclerView` to onResume as well when you do this.